src/transforms/interval.js

```diff
@@ -28,7 +29,9 @@ function maybeIntervalK(k, maybeInsetK, options, trivial) {
   let D1, V1;
   function transform(data) {
     if (V1 !== undefined && data === D1) return V1; // memoize
-    return (V1 = map(valueof((D1 = data), value), (v) => interval.floor(v)));
+    let V = valueof((D1 = data), value);
+    if (isTimeInterval(interval)) V = map(V, coerceDate, Float64Array);
```

Comment:
```suggestion
    if (isTimeInterval(interval)) V = coerceDates(V);
```

I think we want the slower `coerceDates` here. This code using Float64Array is faster, but we can only use it in the bin transform because we know it’s not exposed outside the bin transform. I don’t think that’s true here; we want to pass the interval Date instances, not numbers.